File: easy/Stepwise-Word/Stepwise-Word.cs
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
                ShoWord(line);
            // do something with line
        }
    }
    
    public static void ShoWord(string line){
        string[] words = line.Split(' ');
        int index = 0;
        for (int i=0;i<words.Length;i++)if (words[i].Length>words[index].Length) index = i;
        string word = words[index];
        int leng = word.Length;
        for(int i=0;i<leng;i++){
            for(int j=0;j<i+1;j++){
                char ch = '*';
                if (i==j) ch = word[j];
                System.Console.Write(ch);
            }
            System.Console.Write(" ");
        }
        System.Console.WriteLine();
    }
}

File: easy/Stepwise-Word/Stepwise Word.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShoWord(line);
        }
    }
    
    public static void ShoWord(String line){
        String[] words = line.split(" ");
        int index = 0;
        for (int i=0;i<words.length;i++)if (words[i].length()>words[index].length())index = i;
        String word = words[index];
        int leng = word.length();
        for(int i=0;i<leng;i++){
            for(int j=0;j<i+1;j++){
                char ch = '*';
                if (i==j) ch = word.charAt(j);
                System.out.print(ch);
            }
            System.out.print(" ");
        }
        System.out.println();
    }
}
